### ISSUE TITLE
SQL Editor erases state briefly post page-load

### ISSUE THREAD

**Issue by zachblume** [supabase/supabase#22187]
# Bug report

- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

The SQL editor on the cloud platform begins loading, renders a textbox for editing, and begins to scroll the previous query list. About 1 second later, it drops the state of the textbox and re-renders it.

## To Reproduce

1. Load the SQL editor
2. Begin typing
3. Your state is lost approx 1second after pageload

## Expected behavior

The textbox's state should never be dropped after the user has made input

## Screenshots

Screen recording:
https://drive.google.com/file/d/1inG6r90jJf97jl2nFDaBs7oegUc4fQNq/view?usp=sharing

## System information

- OS: Mac 14.0 (23A344)
- Browser (if applies) Chrome Version 123.0.6312.58 (Official Build) (arm64)
- Version of supabase-js: [e.g. 6.0.2]
- Version of Node.js: [e.g. 10.10.0]

**Comment by Hallidayo** [comment]
Hi @zachblume - Are you finding this is happening all the time for you? Just trying my end and I can't seem to get any sort of lag.

**Comment by zachblume** [comment]
Yup, tried it again right now, upgraded Chrome, same thing, every time after page load or refresh. Here is another screen recording with the react tools on showing the render highlight and timing
https://drive.google.com/file/d/1At6D5DkfexZ7klKLKQh8ib0kmZtfvKYN/view?usp=sharing

**Comment by saltcod** [comment]
The SQL Snippets have changed quite a bit since this report. Apologies for missing it. Going to mark this as closed, but feel free to reopen if you're still experiencing this.
